src/plugins/jartic/archive_downloader.py

```
@@ -0,0 +1,241 @@
+                    raise Exception(f"Failed to fetch index: HTTP {response.status}")
+
+                html = await response.text()
+
+            soup = BeautifulSoup(html, 'html.parser')
+
+            archives = []
+            archive_pattern = re.compile(r'(\d{4})_(\d{2})\.zip')
+
+            for link in soup.find_all('a', href=True):
+                href = link.get('href', '')
+                text = link.get_text(strip=True)
```

Ruff (0.12.2)

Local variable `text` is assigned to but never used

Remove assignment to unused variable `text`

(F841)